buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -29,10 +29,17 @@ timeout(time: 6, unit: 'HOURS') {
     stage('Copyright Check') {
         node ('worker') {
             timestamps {
-                git url: SRC_REPO
-                sh "git fetch --tags --progress origin +refs/pull/*:refs/remotes/origin/pr/*"
-                sh "git checkout --detach ${sha1}"
-                sh 'git fetch origin'
+                checkout changelog: false, poll: false,
+                        scm: [$class: 'GitSCM',
+                            branches: [[name: sha1]],
+                            doGenerateSubmoduleConfigurations: false,
+                            extensions: [[$class: 'CloneOption',
+                                            depth: 0,
+                                            noTags: false,
+                                            reference: '/home/jenkins/openjdk_cache',
+                                            shallow: false]],
```

Comment:
Should you add a scm timeout greater than 10 mins. (=default)?

Reply:
Since we are using a reference repository I don't think increasing more than the default 10 minutes is necessary. I've never seen one timeout.